docs/LwM2M_Client/STMicroelectronics/Runtime_certificate.md

```diff
@@ -0,0 +1,94 @@
+# Runtime Certificate and Private Key Configuration
+
+## Introduction
+Anjay library supports secure connection between device and server. Encryption can be achieved with the use of Pre-Shared Key as was described in [STM32L496G-DISCO/BG96](STM32L496G-DISCO/BG96.md), [B-L462E-CELL1/TYPE1SC](B-L462E-CELL1/TYPE1SC.md), [B-L475E-IOT01A](B-L475E-IOT01A.md), and [B-U585I-IOT02A/BG96](B-U585I-IOT02A/BG96.md) pages or with certificate-based cryptography.
```

Comment:
Do these links work? building this page produces warnings:
```
WARNING  -  Documentation file 'LwM2M_Client/STMicroelectronics/Runtime_certificate.md' contains a link to
            'LwM2M_Client/STMicroelectronics/STM32L496G-DISCO/BG96.md' which is not found in the
            documentation files.
WARNING  -  Documentation file 'LwM2M_Client/STMicroelectronics/Runtime_certificate.md' contains a link to
            'LwM2M_Client/STMicroelectronics/B-L462E-CELL1/TYPE1SC.md' which is not found in the
            documentation files.
WARNING  -  Documentation file 'LwM2M_Client/STMicroelectronics/Runtime_certificate.md' contains a link to
            'LwM2M_Client/STMicroelectronics/B-U585I-IOT02A/BG96.md' which is not found in the
            documentation files.
```
and I get 404 on clicking them